Formula/g/gpac.rb

```diff
@@ -31,27 +40,72 @@ class Gpac < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "3fa932d0f671a1643b20f381bba3181328431e5ef9555444a8d25a983ace36e1"
   end
 
+  depends_on "cmake" => :build
   depends_on "pkgconf" => :build
+  depends_on "a52dec"
+  depends_on "aom"
+  depends_on "faad2"
+  depends_on "ffmpeg"
+  depends_on "freetype"
+  depends_on "jpeg"
+  depends_on "kvazaar"
+  depends_on "libcaca"
+  depends_on "libogg"
+  depends_on "libpng"
+  depends_on "libvorbis"
+  depends_on "mad"
+  depends_on "openjpeg"
   depends_on "openssl@3"
+  depends_on "sdl2"
+  depends_on "theora"
+  depends_on "x264"
+  depends_on "xvid"
 
   uses_from_macos "zlib"
 
+  resource "deps_unix" do
+    url "https://github.com/gpac/deps_unix.git", branch: "master"
+  end
+
   def install
-    args = %W[
-      --disable-wx
-      --disable-pulseaudio
-      --prefix=#{prefix}
-      --mandir=#{man}
-      --disable-x11
-    ]
+    (buildpath/"gpac_public").install Dir["*"]
+    (buildpath/"deps_unix").install resource("deps_unix")
 
-    system "./configure", *args
-    system "make"
-    system "make", "install"
+    cd "deps_unix" do
+      ENV["CMAKE_POLICY_VERSION_MINIMUM"] = "3.5"
+      system "./build_all.sh", "osx" if OS.mac?
+      system "./build_all.sh", "linux" if OS.linux?
+    end
+
+    cd "gpac_public" do
+      args = %W[
+        --prefix=#{prefix}
+        --mandir=#{man}
+      ]
+
+      system "./configure", *args
+      system "make", "-j", ENV.make_jobs.to_s
+      system "make", "install"
+    end
+  end
+
+  def caveats
+    <<~EOF
+      GPAC will store its configuration files at:
+        ~/.gpac
+
+      If you wish to remove these files, you can do so with:
+        rm -rf ~/.gpac
+    EOF
   end
```

Comment:
Don't use `caveats` for informing about the storage of configuration files in the home directory. [`caveats` should only be used for informing users about specific issues/gotchas introduced by Homebrew packaging](https://docs.brew.sh/Formula-Cookbook#caveats).